xml/System.Diagnostics.Metrics/Meter.xml

```diff
@@ -466,13 +466,21 @@ Example uses for ObservableCounter: The number of page faults for each process.
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="name">To be added.</param>
-        <param name="observeValues">To be added.</param>
-        <param name="unit">To be added.</param>
-        <param name="description">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="name">The instrument name. Cannot be <see langword="null" />.</param>
+        <param name="observeValues">The callback to call to get the measurements when the <see cref="M:System.Diagnostics.Metrics.ObservableCounter`1.Observe" /> is called by <see cref="M:System.Diagnostics.Metrics.MeterListener.RecordObservableInstruments" />.</param>
```

Comment:
@gewarren do you see anything wrong with this cref?:

```
Cross reference not found: 'System.Diagnostics.Metrics.ObservableCounter`1.Observe'
```

Reply:
@carlossanlop ``ObservableCounter<T>.Observe`` is a protected method in a sealed class - so it can only be called by ObservableCounter<T> right? Should this cref be ``<see cref="M:System.Diagnostics.Metrics.ObservableInstrument`1.Observe" />`` instead?

Reply:
@tarekgh  @noahfalk can you please chime in? Was it intended to mention `ObservableCounter` from a public comment?

Reply:
It supposes be  `<see cref="M:System.Diagnostics.Metrics.ObservableUpDownCounter`1.Observe" />`. I'll try to fix that in the sources.

> ObservableCounter<T>.Observe is a protected method in a sealed class - so it can only be called by ObservableCounter right? 

This doc is about the method `CreateObservableUpDownCounter`, so we should refer to ObservableUpDownCounter.Obseve and not the abstracted instrument one.

Reply:
```suggestion
        <param name="observeValues">The callback to call to get the measurements when the <see cref="M:System.Diagnostics.Metrics.ObservableUpDownCounter`1.Observe" /> is called by <see cref="M:System.Diagnostics.Metrics.MeterListener.RecordObservableInstruments" />.</param>
```

Reply:
@tarekgh does the suggestion look good to you? Do we need to do the same suggestion for lines 518 and 566 of this file?

Reply:
The suggestion looks good to me. We need to apply this suggestion on all `CreateObservableUpDownCounter` doc instances. Thanks for fixing that!